app/models/announcement.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
```

Comment:
It's lacking the usual schema description other models have.

Reply:
It stopped auto-generating them for some reason

Reply:
Fixed. They changed config.